change/@fluentui-eslint-plugin-bdc15f5d-ef63-4e26-89c5-6983d11bfd6a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Revert usage of @rnx-kit/eslint-plugin",
+  "packageName": "@fluentui/eslint-plugin",
+  "email": "andredias@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
In the future please manually update to `dependentChangeType: none` when changing eslint-plugin. Using `patch` causes unnecessary bumps of every package in the repo. (This would also be a good thing to add a feature for in beachball if anyone has time.)